docs/framework/How-to-Get-Updates/How-to-get-updates

```diff
@@ -0,0 +1,18 @@
+## How to Get Security and Quality Rollup .NET Framework updates:
```

Comment:
I think this page shouldn't be about acquisition but about the updates generally, with one or more sections on acquisition.

I'd expect to see the following:

- Description of each update type.
- Schedule of release
- Commulative model
- OSes we support
- Special information for specific versions, like how updates interact with .NET Framework 3.5.